my_list.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'my_enumerable'
+
+class Mylist
+  include Enumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  attr_reader :list
+
+  def each
+    yield @list
+  end
+end
```

Comment:
- [ ] Your class needs to implement `#each` method that yields all members of `@list` in succession. Similar:

```ruby
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```